server.ts

```diff
@@ -17,20 +17,77 @@ const dbConfig = {
   connectionString: process.env.DATABASE_URL,
   ssl: sslSetting,
 };
-
+//hello
 const app = express();
 
 app.use(express.json()); //add body parser to each following route handler
 app.use(cors()) //add CORS support to each following route handler
 
+
 const client = new Client(dbConfig);
 client.connect();
 
+// app.use(cors({
+//   origin: '*',
+// methods: ["GET", "POST","PATCH","PUT"]
+// }));
+
+
 app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
+  res.set('Access-Control-Allow-Origin', '*')
+  const dbres = await client.query('select sub_breed, sum(vote_count) as sumvote_count from vote  group by sub_breed order by sumvote_count desc limit 10');
```

Comment:
could be consistent with query for example on line 61  the clauses are in uppercase and here the query is all in lowercase